requirements.txt

```diff
@@ -0,0 +1,8 @@
+cryptography>=2.1.4 # BSD/Apache-2.0
```

Comment:
It does make sense to start using a requirements.txt, been meaning to do this for a while. We will need to do some updates to add `pip3 install -r requirements`  to the readme and  installer.sh though as well:

https://github.com/keylime/keylime/blob/f4f53fad7b3058da8d04040a454d4d31eae6b473/installer.sh#L598

For the readme you will see those with a <ctr-f> "setup.py" (I can't permalink readme files as github renders them)